patternfly/patternfly-elements
# feat(tabs): PF1:1

## What I did
- removed all scss styling 
- implemented Patternfly v4 styles and API
- improved keyboard navigation, alignment with [W3C aria authoring practices for tabs](https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-automatic.html)

### TODO:
- [x] break out base classes for inheritance
- [x] implement overflow styles 
- [x] implement inset styles 
- [x] implement icon tab styles
- [x] implement filled styles
- [x] fix spec tests
- [x] improve docs (add CSS props jsdoc comments)

### Choices:
for now, does not implement 
- horizontal overflow dropdown feature
- sub tabs feature
- nav element feature
- separate content (trigger) feature
- child tab-panel mounting features
- dynamic closable tabs feature
- loading a tab via an external toggle

## Testing Instructions
- Load up the deploy preview for this branch
- Load up [PatternFly v4 Tabs](https://www.patternfly.org/v4/components/tabs/)
- Compare and contrast
- Read over the [docs](https://deploy-preview-2099--patternfly-elements.netlify.app/components/tabs/) and see if anything's missing or no longer relevant

## Notes to reviewers
This PR is *BREAKING*. It aligns pfe-tabs with pfv4.
